client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -5,55 +5,36 @@
  */
 import React from 'react';
 import { connect } from 'react-redux';
-import Dispatcher from 'dispatcher';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
-import { isEmpty, noop, every, flow, has, defer, get, trim, sortBy, reverse } from 'lodash';
+import { isEmpty, noop, flow, has, get, trim, sortBy, reverse } from 'lodash';
 import url from 'url';
 import moment from 'moment';
-import { stringify } from 'qs';
 
 /**
  * Internal dependencies
  */
-import wpLib from 'lib/wp';
 import config from 'config';
+import wpLib from 'lib/wp';
 import { validateImportUrl } from 'lib/importers/url-validation';
-
-const wpcom = wpLib.undocumented();
-
-import { toApi, fromApi } from 'lib/importer/common';
-
-import {
-	mapAuthor,
-	startMappingAuthors,
-	startImporting,
-	createFinishUploadAction,
-} from 'lib/importer/actions';
-import user from 'lib/user';
-
-import { appStates } from 'state/imports/constants';
-import ErrorPane from '../error-pane';
 import TextInput from 'components/forms/form-text-input';
 import FormSelect from 'components/forms/form-select';
-
-import SiteImporterSitePreview from './site-importer-site-preview';
-
-import { prefetchmShotsPreview } from 'lib/mshots';
-
 import { recordTracksEvent } from 'state/analytics/actions';
-
 import { setSelectedEditor } from 'state/selected-editor/actions';
-
+import {
+	importSite,
+	validateSiteIsImportable,
+	resetSiteImporterImport,
+	setValidationError,
+} from 'state/imports/site-importer/actions';
 import ImporterActionButton from 'my-sites/importer/importer-action-buttons/action-button';
 import ImporterCloseButton from 'my-sites/importer/importer-action-buttons/close-button';
 import ImporterActionButtonContainer from 'my-sites/importer/importer-action-buttons/container';
+import ErrorPane from '../error-pane';
+import SiteImporterSitePreview from './site-importer-site-preview';
+
+const wpcom = wpLib.undocumented();
```

Comment:
There's no need to call `undocumented()` here as no methods from `Undocumented` are used. There are only low-level calls to send GET or POST requests. Then the following is sufficient:
```js
import wpcom from 'lib/wp';

wpcom.req.get( ... );
wpcom.req.post( ... );
```